public/locales/en-US.json

```diff
@@ -319,6 +324,7 @@
     "label_groupVariant_RHEL for x86": "RHEL for x86",
     "label_groupVariant_rhel-for-x86-eus": "RHEL for x86 EUS",
     "label_groupVariant_rhel-for-x86-ha": "RHEL for x86 HA",
+    "label_groupVariant_rhel-for-x86-rs": "RHEL for x86 Resilient Storage",
```

Comment:
this should be ok, albeit huge by comparison to the others... 

for the notes... we were requested to also expand On-Demand for `payg`. but can't to hurt verify with pm and/or docs... we might need to go back and review expanding the others

Reply:
Ok, short term we're leaving this as is